Sources/Testing/Testing.docc/Documentation.md

```diff
@@ -35,10 +35,8 @@ their problems.
 
 #### Related videos
 
-@Links(visualStyle: compactGrid) {
-  - <doc://com.apple.documentation/videos/play/wwdc2024/10179>
-  - <doc://com.apple.documentation/videos/play/wwdc2024/10195>
-}
+- [Meet Swift Testing](https://developer.apple.com/videos/play/wwdc2024/10179)
```

Comment:
Can you explain why this particular change was necessary?

Reply:
This is because the swift-docc-plugin does not support links of this style. It’s off topic for this repo, but platform vendors which distribute Swift Testing can choose to revert this specific change if they wish.